Source/JavaScriptCore/llint/WebAssembly.asm

```diff
@@ -777,10 +769,11 @@ else
     storep ws1, constexpr (CallFrameSlot::callee - CallerFrameAndPC::sizeInRegisters) * 8 + TagOffset[sp]
 end
 
-    loadp Wasm::JSEntrypointCallee::wasmFunctionPrologue[ws0], ws0
-    call ws0, WasmEntryPtrTag
+    call Wasm::JSEntrypointCallee::wasmFunctionPrologue[ws0], WasmEntryPtrTag
 
+if ASSERT_ENABLED
     clobberVolatileRegisters()
+end
```

Comment:
Why remove the debug assertion? Perf regression?

Reply:
It was not intended to make this always. So just fixing it.